Source/WebCore/page/ProcessSyncData.in

```diff
@@ -58,3 +58,5 @@
 
 MainFrameURLChange : URL [Header=<wtf/URL.h>]
 AudioSessionType : WebCore::DOMAudioSessionType [DocumentSyncData Conditional=ENABLE(DOM_AUDIO_SESSION) Header="DOMAudioSession.h"]
+IsAutofocusProcessed : bool [DocumentSyncData]
```

Comment:
I think it would be a little nicer to have the attributes before the other stuff, but no strong opinion.